dashboards/aws-apprunner/aws-apprunner.json

```diff
@@ -0,0 +1,208 @@
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` = '/aws/apprunner/${AppRunner-container-name}/${application-hash}/application' SINCE 2 hours ago facet `aws.logGroup` TIMESERIES MAX"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Application logs",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "logger.log-table-widget"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT `log_severity`,`timestamp`,`message` FROM Log WHERE `aws.logGroup` = '/aws/apprunner/${AppRunner-container-name}/${application-hash}/application'"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Service logs",
+          "layout": {
+            "column": 9,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` = '/aws/apprunner/${AppRunner-container-name}/${application-hash}/service' SINCE 2 hours ago facet `aws.logGroup` TIMESERIES MAX"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Service logs",
+          "layout": {
+            "column": 9,
+            "row": 4,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "logger.log-table-widget"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT `log_severity`,`timestamp`,`message` FROM Log WHERE `aws.logGroup` = '/aws/apprunner/${AppRunner-container-name}/${application-hash}/service'"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Metrics (Active Instances)",
+          "layout": {
+            "column": 1,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(`aws.apprunner.ActiveInstances`) FROM Metric WHERE `collector.name` = 'cloudwatch-metric-streams' AND `aws.accountId` = '533243300146' SINCE 24 HOURS AGO TIMESERIES"
```

Comment:
Will this aws account id be the same for every single user that installs this dashboard? If not, we're going to need to find a way to query for this data without using a specific account id.